CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Contributions Catalog login callback endpoint to /catalog/auth/callback. [#803](https://github.com/rokwire/rokwire-building-blocks-api/issues/803) 
 
 ### Fixed
+- Added additional env variable for audience for ROKWIRE_AUTH_HOST tokens. [#842](https://github.com/rokwire/rokwire-building-blocks-api/issues/842)
```

Comment:
One minor suggestion. To keep the chronological order of changes in the CHANGELOG, could you please move this entry to the bottom of the Fixed section? Thanks.

Reply:
Sure. Thanks for the suggestion.